### DISCUSSION TITLE
What is the content recommended by the for you feed related to?

### DISCUSSION THREAD

**Discussion by Jxpro** [community/community#13327] [Other Feature Feedback, Questions, & Ideas]
I'm wondering, I didn't follow this person, and I didn't star his project, and I didn't even remember ever cooperating with him.

Why do I have his card here? 

How do I know what contact I've had with him?

Or is there any way for me to not show his card besides blocking him?

![uTools_164803720836](https://user-images.githubusercontent.com/87376573/159695497-e1f38923-bded-42e8-9191-668a6f1d6de5.png)

![uTools_164803722448](https://user-images.githubusercontent.com/87376573/159695516-c77894f9-aaf7-471c-a738-436ef03b91c7.png)

**Comment by Jxpro** [comment]
Can anyone help me ?
Thanks a lot !

**Comment by cmtheit** [comment]
I don't know. But I would support  your question's spreading.

**Reply by Jxpro** [reply]
Thanks for your help.

**Comment by likwid** [comment]
Did you work with the person in a previous job, or were you both part of the same organization at some point?  I ask because I also have someone showing up on my feed, despite me never starring their repositories, or following them.  We did work at the same place, with some small overlap (a few months).

**Reply by Jxpro** [reply]
I joined an organization but haven't been involved in development yet, and he's not here right now. But I don't know how to check previous members of the organization. So I'm not sure if he's ever been in an organization with me before. Could you please tell me how can I know? In addition, I have submitted issues in other projects, but have not submitted PR. I don't know if the issue will be considered as cooperation.

**Comment by Jxpro** [comment]
Or is there any way for me to not show his card besides blocking him?

**Reply by JohnSThomas** [reply]
They definitely need a not relevant button, or a don't show in my feed option.

**Reply by saintaardvark** [reply]
I would _loooooove_ something like this.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
